.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ jobs:
     outputs:
       style: ${{ steps.filter.outputs.style }}
       unit-tests: ${{ steps.filter.outputs.unit-tests }}
+      coverage: ${{ steps.filter.outputs.coverage }}
       container: ${{ steps.filter.outputs.container }}
```

Comment:
```suggestion
```
style & unit-tests are the only filters defined in the paths-filter step, so we can actually delete both of these since they only serve to forward on the results from the filter step. (I think I accidentally left in the container bit from Hubcast / Gantry.)